### PR TITLE
godb: add existing godb sources

### PR DESCRIPTION
**Pull request by mgenov**
Added all existing godb sources.

Here is a small list of changes that are performed:
 * the package name is changed from datastore to godb
 * testdb host is specified using env variable TEST_DB_HOST

### CODE REVIEW

File: datastoretest/mongo.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	"strings"
+
+	"gopkg.in/mgo.v2"
+
+	"github.com/clouway/godb"
+	"github.com/clouway/godb/mongo"
+)
+
+const (
+	host = "dev.telcong.com"
```
Comment (mgenov) [inline datastoretest/mongo.go]: This need to be changed.

Comment (mgenov) [reply datastoretest/mongo.go]: It's changed to use environment variable instead.

**Comment by EmilGeorgiev** [comment]
LGTM

**Review by EmilGeorgiev — APPROVED** [review]

**Comment by FeNoMeNa** [comment]
LGTM

**Pull request merged**